main/gradebook/lib/be/category.class.php

```diff
@@ -948,9 +948,10 @@ public function calc_score(
                 foreach ($links as $link) {
                     $link->setStudentList($this->getStudentList());
 	                
-	                //Dont forget to add session id
-	                if($session_id) $link->set_session_id($session_id);
-	
```

Comment:
You should configured your code editor in order to use "4 spaces" (instead of tabs) for spacing.